creation/lib/cvWParamDict.py

```diff
@@ -1100,9 +1097,21 @@ def populate_common_descript(descript_dict, params):
     descript_dict.add("FactoryCollectors", repr(collectors))
 
     schedds = []
+    valid_schedd = False
+    undefined_schedds = 0
     for el in params.match.job.schedds:
-        cWDictFile.validate_node(el["fullname"])
+        # A single submit host not in the DNS should not fail the reconfig
+        # Especially in production there are many submit hosts, some are temporary nodes
+        # Would be useful to have a WARNING message, but the current implementation allows only fail/continue
+        # Still raising an invalid configuration exception if no schedd is in DNS
+        try:
+            cWDictFile.validate_node(el["fullname"], check_dns=False)
+            valid_schedd = True  # skipped if exception is risen
+        except RuntimeWarning:
+            undefined_schedds += 1
         schedds.append(el["fullname"])
+    if undefined_schedds > 0 and not valid_schedd:
+        raise RuntimeError("No valid schedd found, all are unknown to DNS")
```

Comment:
Having `valid_schedd` and `undefined_schedds` here seems redundant to me. If we don't need the exact number of undefined schedds, `valid_schedd` seems to be enough.